src/nav2_msgs/srv/GetCostmap.srv

```diff
@@ -0,0 +1,6 @@
+# Get the costmap
+
+# Specifications for the requested costmap
+nav2_msgs/Costmap map
+---
+nav2_msgs/Costmap map
```

Comment:
newline.

Reply:
Fixed. Also realized that the request should be nav2_msgs/CostmapMetaData -- no need for the data field.